lang/po/Firmware_fr.po

```diff
@@ -601,87 +611,92 @@ msgid "Checking Y axis"
 msgstr "Verification axe Y"
 
 # MSG_SELFTEST_CHECK_Z c=20
-#: 
+#: ultralcd.cpp:8501
 msgid "Checking Z axis"
 msgstr "Verification axe Z"
 
 # MSG_CHOOSE_EXTRUDER c=20 r=1
-#: messages.c:52
+#: messages.c:53
 msgid "Choose extruder:"
 msgstr "Choisir extrudeur:"
 
 # MSG_CHOOSE_FILAMENT c=20 r=1
-#: messages.c:53
+#: messages.c:54
 msgid "Choose filament:"
 msgstr "Choix du filament:"
 
 # MSG_FILAMENT c=17 r=1
-#: messages.c:33
+#: messages.c:34
 msgid "Filament"
 msgstr ""
 
 # MSG_WIZARD_XYZ_CAL c=20 r=8
-#: ultralcd.cpp:4956
+#: ultralcd.cpp:5059
 msgid "I will run xyz calibration now. It will take approx. 12 mins."
 msgstr "Je vais maintenant lancer la calibration XYZ. Cela prendra 12 min environ."
 
 # MSG_WIZARD_Z_CAL c=20 r=8
-#: ultralcd.cpp:4964
+#: ultralcd.cpp:5067
 msgid "I will run z calibration now."
 msgstr "Je vais maintenant lancer la calibration Z."
 
 # MSG_WATCH
-#: messages.c:114
+#: messages.c:116
 msgid "Info screen"
 msgstr "Ecran d'info"
 
 # MSG_INSERT_FILAMENT c=20
-#: ultralcd.cpp:2564
+#: ultralcd.cpp:2591
 msgid "Insert filament"
 msgstr "Inserez le filament"
 
 # MSG_FILAMENT_LOADED c=20 r=2
-#: messages.c:36
+#: messages.c:37
 msgid "Is filament loaded?"
 msgstr "Fil. est-il charge?"
 
 # MSG_STEEL_SHEET_CHECK c=20 r=2
-#: messages.c:107
+#: messages.c:109
 msgid "Is steel sheet on heatbed?"
 msgstr "Est la plaque sur le plat. chauffant?"
 
 # MSG_LAST_PRINT_FAILURES c=20
-#: messages.c:55
+#: messages.c:56
 msgid "Last print failures"
 msgstr "Echecs derniere imp."
 
+# MSG_WIZARD_WELCOME_SHIPPING c=20 r=16
+#: messages.c:122
+msgid "Hi, I am your Original Prusa i3 printer. I will guide you through a short setup process, in which the Z-axis will be calibrated. Then, you will be ready to print."
+msgstr "Bonjour, je suis votre imprimante Original Prusa i3. Je vais vous accompagner au cours d'un bref processus de reglage, qui permettra de calibrer le Z-axis. Apres cela, tout sera pret pour imprimer."
```

Comment:
```suggestion
msgstr "Bonjour, je suis votre imprimante Original Prusa i3. Je vais vous accompagner au cours d'un processus bref de reglage pour calibrer le Z-axis. Apres cela, tout sera pret pour imprimer."
```

##### `bref processus` -> `processus bref`
`bref` is describing the length, not the size of the process, and should come after the noun.

##### `processus de relage, qui permettra de calibrer` -> `processus de relage pour calibrer`
`qui permettra de calibrer` sounds like you're saying `One of the things you can do in this calibration process is calibrate the Z-axis, but it's not necessary for printing`. If the process is to calibrate the Z-axis and it's necessary to print, then you should say `pour calibrer`.